client/my-sites/pages/page/index.js

```diff
@@ -48,13 +49,13 @@ function recordEvent( eventAction ) {
 }
 
 function getReadableStatus( status ) {
-	var humanReadableStatus = {
-		'publish': i18n.translate( 'Published' ),
-		'draft': i18n.translate( 'Draft' ),
-		'pending': i18n.translate( 'Pending' ),
-		'future': i18n.translate( 'Future' ),
+	const humanReadableStatus = {
+		publish: i18n.translate( 'Published' ),
```

Comment:
Since the component is already `localize`d, can't we use `props.translate` here and remove the `i18n` import above?